apps/web/components/shared/productvariant.tsx

```
@@ -0,0 +1,430 @@
+	Bookmark,
+	Gift,
+	Globe,
+	Heart,
+	Share2,
+	Star,
+	Trophy,
+} from 'lucide-react'
+import type React from 'react'
+import { useEffect, useState } from 'react'
+
+type ProjectCardProps = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)